esphome/components/sht3xd/sensor.py

```diff
@@ -38,6 +38,11 @@
             ),
         }
     )
+    .extend(
+        {
+            cv.Required("heater_enabled", default=True): cv.boolean,
+        }
+    )
```

Comment:
This can go directly inside the schema, and also needs to define `CONF_HEATER_ENABLED = "heater_enabled"` higher up in the file
```suggestion
        },
        cv.Optional(CONF_HEATER_ENABLED, default=True): cv.boolean,
    )
```